en/config/autotune.md

```diff
@@ -131,6 +143,21 @@ This behaviour can be configured using the [MC_AT_APPLY](../advanced_config/para
   If the control loop is unstable, the control gains are immediately reverted back to their previous value.
   If the test passes, the pilot can then use the new tuning.
 
+### Enable/Disable Autotune Switch (Fixed Wing)
+
+A remote control switch can be configured to enable/disable autotune (in any mode) using an RC AUX channel.
+
+To map a switch:
+
+1. Select an RC channel on your controller to use for the autotune enable/disable switch.
+1. Set [RC_MAP_AUX1](../advanced_config/parameter_reference.md#RC_MAP_AUX1) to match the RC channel for your switch (you can use any of `RC_MAP_AUX1` to `RC_MAP_AUX6`).
+1. Set [FW_AT_MAN_AUX](../advanced_config/parameter_reference.md#FW_AT_MAN_AUX) to the selected channel (i.e. `1: Aux 1` if you mapped `RC_MAP_AUX1`).
+
+The auto tuner will be disabled when the switch is below `0.5` (on the manual control setpoint range of of `[-1, 1]` and enabled when the switch channel is above `0.5`.
```

Comment:
Note, you had some info about the possible switch end points. I don't _think" that was needed, in particular because switch values are likely to vary/be configurable.

Reply:
@hamishwillee Yeah this is where I need help determining what level of information is necessary to get the point across. I don't know if it's missing information that when an RC channel is mapped to an aux, it's end point is translate from 1000-2000 to [-1, 1], so the point here is that a user needs to make sure their switch has endpoints correctly configured. What do you think?

Reply:
@chris1seto I don't know what the end points refer to so I can't answer.

My thinking
- an RC switch with two switch positions will output different PWM values in each. 
- A channel has some range it can output, and the switch position values will fall between those.
  - Is this what you mean by "end points" - the range of possible values in the channel?
  - Does the channel range value vary by radio or are you saying it is always  1000-2000 us?
- If PX4 maps the range to -1 and +1 it must have some way of knowing/setting the input range?

I think you are saying that we can make the end points such that the two values actually emitted by the switch are on either side of the mapped +0.5 value.